src/MassUpdatable.php

```diff
@@ -20,6 +20,22 @@
  */
 trait MassUpdatable
 {
+    public static function withoutMassUpdateTimestamps(): self
+    {
+        return tap(new static(), function (self $model) {
+            $model->timestamps = false;
+        });
+    }
+
+    public function scopeMassUpdateQuietly(Builder $query, array | Arrayable $values, array | string | null $uniqueBy = null): int
+    {
+        $this->timestamps = false;
+
+        return tap($query->massUpdate($values, $uniqueBy), function () {
+            $this->timestamps = true;
+        });
+    }
+
```

Comment:
```suggestion
    public function scopeMassUpdateQuietly(Builder $query, array | Arrayable $values, array | string | null $uniqueBy = null): int
    {
        return static::withoutTimestamps(fn() => $query->massUpdate($values, $uniqueBy));
    }
```

I'm pretty sure this is all that is needed now, @cwilby! Don't take my word for it!

Reply:
I think this suggestion works for people on Laravel ^9.31, but it will break on older versions (I still have support for ^8.71 in place)

Reply:
So, the original design of this single method would be best until the minimum package support reaches 9.31?

Reply:
FWIW - I don't think the `withoutMassUpdateTimestamps` method is needed in the library. I think there should be a `massUpdate` and a `massUpdateQuietly` method. I think it solves the ergonomic concerns I originally raised.

Reply:
I have removed `withoutMassUpdateTimestamps`, leaving `massUpdateQuietly` as the only addition.

Internally, `massUpdateQuietly` does not use the L9 `withoutTimestamps` method, which if I've understood correctly, should provide a backward compatible solution.